tsconfig.json

```diff
@@ -76,5 +80,27 @@
     "src/env.js",
     "src/lib/hofs/**/*.js",
     "src/lib/guards.js",
+    "src/files/file-preview/file-thumbnail.tsx",
+    "src/files/type-from-ext/index.js",
+    "src/files/files-grid/files-grid.tsx",
+    "src/files/files-grid/grid-file.tsx",
+    "src/files/file-icon/FileIcon.js",
+    "src/files/pin-icon/PinIcon.js",
+    "src/icons/GlyphDots.js",
+    "src/components/checkbox/Checkbox.js",
+    "src/icons/GlyphPin.js",
+    "src/icons/GlyphPinCloud.js",
+    "src/icons/GlyphSmallTick.js",
+    "src/icons/GlyphDocText.js",
+    "src/icons/GlyphDocPicture.js",
+    "src/icons/GlyphDocMusic.js",
+    "src/icons/GlyphDocMovie.js",
+    "src/icons/GlyphDocGeneric.js",
+    "src/icons/GlyphDocCalc.js",
+    "src/icons/GlyphFolder.js",
+    "src/icons/GlyphPinCloud.js",
+    "src/icons/GlyphPin.js",
+    "src/icons/StrokeCube.js",
+    "src/files/type-from-ext/extToType.js"
```

Comment:
ts needed these files in the project because they're being imported by other files.. we really need to finish up the TS migration in this project because it's getting messy

Reply:
attempt at an explanation is here: https://github.com/ipfs/ipfs-webui/issues/2349